src/aks-preview/azext_aks_preview/decorator.py

```diff
@@ -1759,17 +1760,21 @@ def get_disk_driver(self) -> Optional[ManagedClusterStorageProfileDiskCSIDriver]
                 if disk_driver_version:
                     profile.version = disk_driver_version
             elif disable_disk_driver:
+                msg = "Please make sure there are no existing PVs and PVCs that are used by AzureDisk CSI driver before disabling."
+                if not self.get_yes() and not prompt_y_n(msg, default="n"):
+                    raise DecoratorEarlyExitException()
```

Comment:
Will the execution of CLI scripts in the automation scenario be blocked by such interactive steps that require input?

Reply:
yes. but `-y` can be added into scripts.